src/uct/ib/mlx5/ib_mlx5.inl

```diff
@@ -93,6 +199,10 @@ uct_ib_mlx5_poll_cq(uct_ib_iface_t *iface, uct_ib_mlx5_cq_t *cq)
         return NULL; /* No CQE */
     }
 
+    if (UCT_IB_MLX5_CQE_FORMAT(cqe->op_own) == UCT_IB_MLX5_CQE_COMPRESSED) {
```

Comment:
can we unite this with the 'if' in line 194?
so if CQE is NOT compressed, there will be only one branch

Reply:
Pls see in the code. Hope I understood you correctly.